### PR TITLE
LPS-75265 Refactor Text field to Lexicon version 2

### PR DESCRIPTION
**Pull request by interaminense**

### CODE REVIEW

File: dynamic-data-mapping-type-text/src/main/resources/META-INF/resources/text.soy [hunk 1]
```diff
@@ -11,36 +11,40 @@
  * Prints the DDM form text field.
  */
 {template .render}
+	{@param name: string}
```
Comment (rafaprax) [inline dynamic-data-mapping-type-text/src/main/resources/META-INF/resources/text.soy]: Hey @interaminense Could you please sort the parameter list? Thx

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
@vicnate5 Same tests are failing here

**Comment by vicnate5** [comment]
@leoadb Same regression causing multiple test failures on the subrepo and also blocking the merge to central https://github.com/liferay/liferay-portal/pull/2120#issuecomment-338237600

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
ci:retest

**Pull request merged**